tests/pyminisketch.py

```diff
@@ -0,0 +1,507 @@
+#!/bin/env python3
```

Comment:
Can you switch this to `#!/usr/bin/env python3`

Apparently, that's what [CI wants](https://cirrus-ci.com/task/5751536971153408) for python files included in the bitcoin core repo.

Reply:
https://github.com/sipa/minisketch/pull/32